cmd/snap-bootstrap/initramfs_mounts_state.go

```diff
@@ -48,21 +48,30 @@ type initramfsMountsState struct {
 	recoverySystem string
 
 	verifiedModel gadget.Model
+	seeds         map[string]seed.Seed
 }
 
 var errRunModeNoImpliedRecoverySystem = errors.New("internal error: no implied recovery system in run mode")
 
-// ReadEssential returns the model and verified essential
-// snaps from the recoverySystem. If recoverySystem is "" the
+// LoadSeed returns the seed for the recoverySystem
+// If recoverySystem is "" the
```

Comment:
`fill-paragraph` (M-q) with an appropriate column set with `set-fill-column` (C-x f) would be helpful here :wink:

Reply:
Fixed in 1f71ea9601b3e29e0e88ed52abd4880cf2ebdf80